sdk/storage/azure-storage-blob/downloadtofiledoesnotexist.txt

```diff
@@ -0,0 +1 @@
+default
```

Comment:
? is this required for this PR?

Reply:
From running tests, will make a fix to that test and remove this in another PR